Dashboard/src/EntityDetails/EditEntity.js

```diff
@@ -13,6 +13,13 @@ import EntityTable from "./EntityTable"
  * routed at /EditEntity
  */
 const EditEntity = (props) => {
+  /**
+   * 1) indexesOfSavedData keep records of table data cell indexes
+   * 2) editEntityData is modified data needed to send to next component/pipeline
+   * */
+  const [indexesOfEditEntityData, setIndexesOfEditEntityData] = useState([])
```

Comment:
why do we need indexes? Seems unneccessary

Reply:
indexes=                       [0,       1,   2,     3]
indexesOfSavedData = [10,    6,  14,   22]
editEntityData           = [obj, obj, obj, obj]

remove index 14

indexes=                       [0,       1,   2]
indexesOfSavedData = [10,    6,   22]
editEntityData           = [obj, obj, obj]

1.) To keep track of the order in EditEntityData and get the index to slice off particular EditEntityData.
To show the order of Edit Entity data in Eric component. The helix web app confirm component does this.